faq.md

```diff
@@ -33,27 +28,27 @@ No. Do not post links or downloads to such resources or you will be banned.
 
 * Windows 8 or later
 * 64-bit x86 processor supporting AVX
-* A Vulkan compatible GPU from [this list](http://vulkan.gpuinfo.org/) (NVIDIA recommended)
+* A Vulkan (or D3D12) compatible GPU from [this list](https://vulkan.gpuinfo.org/)
```

Comment:
Probably better to do something like:

A compatible GPU
* Vulkan supporting GPU from this list.
* D3D12 supporting GPU.